site/public/css/calendar.css

```diff
@@ -0,0 +1,273 @@
+ /* the entire table of calendar */
+.table-calendar
+{
+    text-align: left;
+    border:1px solid #000
+}
+
+/* the title at the top of the calendar table, showing month, year */
+.cal-navigation
+{
+    margin-bottom: 0;
+    margin-top: 3px;
+}
+
+.cal-title
+{
+    text-align: center;
+}
+
+.cal-btn {
+    font-size: 28px;
+    padding-left:10px;
+    padding-right:10px;
+    vertical-align: middle;
+}
+
+a.cal-btn {
+    color: var(--btn-text-white);
+}
+
+a.cal-btn:hover {
+    color: var(--btn-text-white);
+    background: rgb(83, 83, 83);
+    border-radius: 20%;
+}
+
+a.cal-btn:visited {
+    color: var(--btn-text-white);
+}
+
+a.cal-btn:focus {
+    color: var(--btn-text-white);
+}
+
+.cal-next-btn {
+    margin-left: 20px;
+}
+
+.cal-prev-btn {
+    margin-right: 20px;
+}
+
+
+#prev-month-switch
+{
+    text-align: right;
+}
+
+#next-month-switch
+{
+    text-align: left;
+}
+
+.cal-month-title
+{
+    font-size: 26px;
+    font-family: 'Source Sans Pro', sans-serif;
```

Comment:
Why is the font family being set for this element here, toggling it on the inspector for mac firefox and chrome, it doesn't appear to do anything so I think it can be removed